client-java/controller/src/main/java/org/evomaster/client/java/controller/mongo/MongoOperation.java

```diff
@@ -0,0 +1,18 @@
+package org.evomaster.client.java.controller.mongo;
+public class MongoOperation {
+    private final Object query;
+    private final Object collection;
+
```

Comment:
I added doc and check. Instead of checking class name I check if it's an implementation of specific class as both MongoCollection and Bson are interfaces.